doc/app/public.md

```diff
@@ -1,13 +1,76 @@
 # Public API for App
 
+## Enum `Status`
+
+The status enumeration contains the following possible values:
+
+* `NOT_REGISTERED`
+* `REGISTERED`
+* `REGISTERED_GOT_MATERIAL`
```

Comment:
I suggest to use "REGISTERED_WITH_MATERIAL"

Reply:
I like that change.